frontend/src/components/auth/Login.jsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState, useEffect } from 'react';
+import Typography from '@material-ui/core/Typography'
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid'
+import { makeStyles } from '@material-ui/core/styles';
+import { useHistory, Redirect } from "react-router-dom";
+import axios from "axios";
+
+const Login = (props) => {
```

Comment:
would it be appropriate to call this `LoginDialog`?

Reply:
Yeah that works for me.

Reply:
Sure that makes sense to me.